bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/discovery/ZonePlayerDiscoveryParticipant.java

```diff
@@ -83,7 +83,8 @@ public Set<ThingTypeUID> getSupportedThingTypeUIDs() {
                 String id = SonosXMLParser
                         .buildThingTypeIdFromModelName(device.getDetails().getModelDetails().getModelName());
                 String udn = device.getIdentity().getUdn().getIdentifierString();
-                if (!id.isEmpty() && !"Sub".equalsIgnoreCase(id) && !udn.isEmpty()) {
+                if (!id.isEmpty() && !SonosBindingConstants.UNSUPPORTED_KNOWN_IDS.contains(id.toLowerCase())
```

Comment:
@lolodomo i would prefer equals above contains, but in that case the constant needs all exact modelnames. Do you know them or have a source where i can find them?

Reply:
No, we discover them each time Sonos introduces a new model...